pkg/api/datasources.go

```diff
@@ -377,7 +378,9 @@ func validateTeamHTTPHeaderJSON(jsonData *simplejson.Json) error {
 	// each teams headers
 	for _, teamheaders := range teamHTTPHeadersJSON {
 		for _, header := range teamheaders {
-			if !contains(validHeaders, header.Header) {
+			if !slices.ContainsFunc(validHeaders, func(v string) bool {
```

Comment:
ohh no way! they have the generics done for slices. nice one

Reply:
Yes, but in experimental package. We use it in few places already.

Reply:
experimental from golang means 99% ready